src/tensors/braidingtensor.jl

```diff
@@ -125,9 +125,9 @@ end
     end
 end
 
-Base.similar(::BraidingTensor, T::Type, P::TensorMapSpace) = TensorMap(undef, T, P)
+# efficient copy constructor
+Base.copy(b::BraidingTensor{E,S}) where {E,S} = BraidingTensor{E,S}(b.V1, b.V2, b.adjoint)
```

Comment:
Since all the field values should be immutable, is this not just `copy(b::BraidingTensor) = b` ? Do we use this somewhere?